### DISCUSSION TITLE
can you have pure html pages in quarto website project?

### DISCUSSION THREAD

**Discussion by carsen-stringer** [quarto-dev/quarto-cli#6679] [Q&A] [answered]
### Description

Can you put html files into your quarto project and have them as links from quarto pages (ideally with the github-pages hosting approach)? I have html files that I've written myself that I didn't want to have to convert, but I can if necessary. Apologies if this is answered elsewhere, thanks for the great tool!

**Comment by cscheid** [comment, marked as answer]
Yes, just use the regular link notation and it should just work. Did that not work for you?

**Reply by carsen-stringer** [reply]
thank you yes it's working in the quarto markdown perfectly as well (instead of linking)
